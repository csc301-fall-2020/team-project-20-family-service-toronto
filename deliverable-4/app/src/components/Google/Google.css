#container {
	height: 100vh;
}

#button-container {
}

#google-iframe {
	position: relative;
	display: initial;
	height: 90vh;
	width: 100vw;
	border: 0;
}

#return-button-div {
	padding: 10px;
}

#return-button-google {
	width: 100%;
	height: 8vh;
	font-size: 150%;
	display: inline-block;
	background: white;
	color: #444;
	border-radius: 50px;
	border: thin solid #888;
	box-shadow: 1px 1px 1px grey;
	white-space: nowrap;
}

iframe {
	overflow: hidden;
}