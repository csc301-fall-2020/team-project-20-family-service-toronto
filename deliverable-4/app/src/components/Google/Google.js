import React, { Component } from "react";
import Iframe from 'react-iframe';
import './Google.css';

export default class Google extends Component {
  
	to_main = () => {
		window.open("/", "_self");
	}

  	render() {
    	return (
      		<div id="container">
        		<Iframe id="google-iframe" data-testid="google-iframe" src="https://www.google.com/webhp?igu=1"/>
        		<div>
        			<button id="return-button"  data-testid="return-button" onClick={() => this.to_main()} type="submit">Return</button>
        		</div>
      		</div>
    	);
  	}
}