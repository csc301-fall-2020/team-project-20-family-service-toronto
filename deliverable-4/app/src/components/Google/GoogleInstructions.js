import React, { Component } from "react";
import './Google.css';
import './GoogleInstructions.css';

export default class GoogleInstructions extends Component {
  
  	to_google = () => {
  		window.open("/Google", "_self");
  	}

    to_main = () => {
      window.open("/", "_self");
    }

  	render() {
    	return (

          <div id="container=fluid" className="inst-cont">
            <div id="row">
              <div id="col-12">
                <img src={"images/laptop_man.png"} alt="google" id="lap-man-photo"></img>
              </div>
              <div id="col-12">
                <header>Instructions</header>
                <ol>
                  <li>Press the <i>Google Application</i> button to go to Google</li>
                  <li>Once in the application, click on the search bar at the top to make a google search</li>
                  <li>You may then scroll through the results by swiping on the screen, clicking on any links that interest you</li>
                  <li>Once ready to exit and go to a different application, you can simply press the <i>Return</i> to come back to this page</li>
                </ol>
              </div>
              <div id="col-12">
                <div id="instructions-buttons">
                  <button className="button shadow p-3 mb-5 bg-white rounded" 
                          id="google-app-button"
                          onClick={() => this.to_google()} 
                          type="submit">Google Application</button>
                  <button className="button shadow p-3 mb-5 bg-white rounded" 
                          id="to-main-button"
                          onClick={() => this.to_main()} 
                          type="submit">Main Page</button>
                </div>
              </div>
            </div>
          </div>
    	);
  	}
}
