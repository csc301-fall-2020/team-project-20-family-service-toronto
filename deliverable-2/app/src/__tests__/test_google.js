import { render, screen } from '@testing-library/react';
import React from 'react';
import ReactDOM from 'react-dom';

import Google from '../components/Google/Google.js';

import { shallow } from 'enzyme';

// test('it calls start logout on button click', () => {
//     const mockLogout = jest.fn();
//     const wrapper = shallow(<Google startLogout={mockLogout}/>);
//     wrapper.find('button').at(0).simulate('click');
//     expect(mockLogout).toHaveBeenCalled();
// });
