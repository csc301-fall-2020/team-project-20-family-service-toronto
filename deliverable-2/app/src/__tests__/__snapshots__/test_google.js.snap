// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders 1`] = `
<div
  id="container"
>
  <iframe
    id="google-iframe"
    src="https://www.google.com/webhp?igu=1"
    style={
      Object {
        "display": "block",
      }
    }
  />
  <div>
    <button
      data-testid="return-button"
      id="return-button"
      onClick={[Function]}
      type="submit"
    >
      Return
    </button>
  </div>
</div>
`;
